### PR TITLE
Update upgrade-distribution-lists.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about Owner

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/manage/upgrade-distribution-lists.md [hunk 1]
```diff
@@ -46,6 +46,10 @@ You must be an Office 365 global admin or Exchange admin to upgrade a distributi
 > [!NOTE]
 > If you're getting the groups digest emails you may notice at the bottom that it will sometimes offer to let you upgrade any eligible distribution lists that you're the owner of. See [Have a group conversation in Outlook](https://support.office.com/article/a0482e24-a769-4e39-a5ba-a7c56e828b22.aspx) for more information about digest emails.
 
+> [!NOTE]
+> The Distribution group must have an owner and the owner must have mailbox in order to upgrade to O365 groups.
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/upgrade-distribution-lists.md]:
```suggestion
> The distribution group must have an owner and the owner must have a mailbox in order to upgrade to O365 Groups.
```

File: Office365-Admin/manage/upgrade-distribution-lists.md [hunk 2]
```diff
@@ -46,6 +46,10 @@ You must be an Office 365 global admin or Exchange admin to upgrade a distributi
...
+
```
Comment (nenonix) [inline Office365-Admin/manage/upgrade-distribution-lists.md]:
```suggestion
```

File: Office365-Admin/manage/upgrade-distribution-lists.md (hunk 1 above)
Comment (nenonix) [inline Office365-Admin/manage/upgrade-distribution-lists.md]:
```suggestion
> To upgrade to Office 365 groups, a distribution group must have an owner with a mailbox. 
```

**Comment by msbemba** [comment]
> Suggest note goes at line 32 and maybe you could add: Get help with [distribution group issues](https://docs.microsoft.com/office365/troubleshoot/administration/distribution-list-issues).... (Or something like that).

Added it to Line 32.

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review based on issue #803

Thank you.

cc: @kenwith

**Pull request merged**